### PR TITLE
Initial commit for badaso-blog plugin

### PR DESCRIPTION
**Pull request by dannyatthaya**
### **Add**
* Add posts.
* Add categories.
* Add tags.
* Add comments.
* Add swagger for API documentation.

### CODE REVIEW

**Comment by rizkiheryandi** [comment]
Please fix the CI first @dannyatthaya

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "version": "1.0.0-alpha.1",
+    "name": "uasoft-indonesia/badaso-blog-module",
+    "description": "Zero development for blog system on badaso",
+    "keywords": ["laravel", "admin", "panel", "dashboard", "pwa", "vue", "generator", "blog", "badaso"],
+    "license": "proprietary",
+    "homepage": "https://badaso.uatech.co.id/",
+    "support": {
+        "issues": "https://github.com/uasoft-indonesia/badaso/issues",
+        "discussion": "https://github.com/uasoft-indonesia/badaso/discussions",
+        "source": "https://github.com/uasoft-indonesia/voyager"
```
Comment (rizkiheryandi) [inline composer.json:11]: wrong source ?

File: src/resources/js/store/modules/badaso.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import api from "../../api";
```
[the remark this answers is not in the document]
Comment (dannyatthaya) [reply src/resources/js/store/modules/badaso.js:1]: Based on offline meetings, badaso team agree to use default store name as "badaso.js".

File: src/resources/js/lang/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
```
[the remark this answers is not in the document]
Comment (dannyatthaya) [reply src/resources/js/lang/index.js:1]: No, this is for detecting components on the plugins itself.

File: src/resources/js/components/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import BadasoCollapse from './BadasoCollapse.vue';
```
[the remark this answers is not in the document]
Comment (dannyatthaya) [reply src/resources/js/components/index.js:1]: No, this is for detecting components on the plugins itself.

**Review by rizkiheryandi — APPROVED** [review]
LGTM

**Pull request merged**